packages/atomic/src/components/search/result-template-components/atomic-quickview-modal/atomic-quickview-modal.tsx

```diff
@@ -83,11 +84,25 @@ export class AtomicQuickviewModal implements InitializableComponent {
   }
 
   private renderHeader() {
-    // TODO: Header should be slottable from result template definition
+    let headerContent: VNode | null = null;
```

Comment:
Is there a specific reason to use null v.s undefined? 🤔

Reply:
Not particularly, just that null is typically used in JSX when you want to render "nothing". 

We need to always render a header for the modal itself, just that it should be empty there is no result to use for the result link.